extensions/wikia/UserProfilePageV3/tests/UserIdentityBoxTest.php

```diff
@@ -31,6 +31,29 @@ public function testCheckIfDisplayZeroStates($data, $expectedResult) {
 		$this->assertEquals($expectedResult, $userIdentityBox->checkIfDisplayZeroStates($data));
 	}
 
+	public function testClearMastheadContents() {
+		/** @var PHPUnit_Framework_MockObject_MockObject|User $userMock */
+		$userMock = $this->getMockBuilder( User::class )
+			->setMethods( [ 'saveSettings' ] )
+			->getMock();
+
+		$userMock->expects( $this->once() )
+			->method( 'saveSettings' )
+			->with();
+
+		$userIdentityBox = new UserIdentityBox( $userMock );
+		$userIdentityBox->clearMastheadContents();
+
+		foreach ( $userIdentityBox->optionsArray as $option ) {
+			if ( $option === 'gender' || $option === 'birthday' ) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "gender" 7 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "birthday" 7 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)